src/main/scala/beam/agentsim/agents/choice/mode/RideHailDefaults.scala

```diff
@@ -10,18 +10,18 @@ import beam.router.model.EmbodiedBeamTrip
   */
 object RideHailDefaults {
   val DEFAULT_COST_PER_MILE = 2.00
-  val zero: Double = 0
+  private val zero: Double = 0D
```

Comment:
nit: I would suggest `0.0` format for Double literals (https://www.scala-lang.org/files/archive/spec/2.13/01-lexical-syntax.html).
Also DEFAULT_COST_PER_MILE (line above) should have type annotation as long as it is public.